pysmt/smtlib/parser.py

```diff
@@ -257,6 +273,36 @@ def _minus_or_uminus(self, *args):
             return mgr.Minus(args[0], args[1])
 
 
+    def _smtlib_underscore(self, *args):
+        """Utility function that handles _ special function in SMTLIB"""
+        mgr = self._current_env.formula_manager
+        if args[0] == "extract":
+            sstart, send = args[1:]
+            try:
+                start = int(sstart.constant_value())
+                end = int(send.constant_value())
+            except ValueError:
+                raise SyntaxError("Expected number in '(_ extract) expression'")
+            return lambda x : mgr.BVExtract(x, end, start)
+
+        if args[0] == "bv0":
+            try:
+                l = int(args[1].constant_value())
+            except ValueError:
+                raise SyntaxError("Expected number in '(_ bv0) expression'")
+            return mgr.BV(0, l)
+        if args[0] == "bv1":
+            try:
+                l = int(args[1].constant_value())
+            except ValueError:
+                raise SyntaxError("Expected number in '(_ bv1) expression'")
+            return mgr.BV((2**l - 1), l)
```

Comment:
My bad, this is WRONG! bv1 in SMTLib means "0000001" not "11111111" (http://swtv.kaist.ac.kr/courses/cs453-fall09/smtlib.pdf).
This should become ```BV(1, l)```